boards/arm/mikroe_clicker_2/mikroe_clicker_2_defconfig

```diff
@@ -0,0 +1,45 @@
+CONFIG_BOARD_MIKROE_CLICKER_2=y
+CONFIG_SOC_SERIES_STM32F4X=y
+CONFIG_SOC_STM32F407XG=y
+# 168MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=168000000
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_USB=y
```

Comment:
As documented in https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations, for  portability purpose, it is application responsibility to enable "non basic" features.
As a consequence, USB should not be enabled here, also please check symbols dependencies (for instance CONFIG_USB_DC_STM32 will be set to y if CONFIG_USB=y).
If needed, USB console can be conditionally configured in board.defconfig.

Reply:
I couldn't find any examples using board.defconfig, I tried making it work in Kconfig.defconfig, but the dependencies were too much to handle.